responder/events.go

```diff
@@ -0,0 +1,109 @@
+package responder
+
+import (
+	"fmt"
+
+	"github.com/flanksource/commons/logger"
+	_ "github.com/flanksource/kommons"
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+	"github.com/flanksource/incident-commander/responder/jira"
+)
+
+func ReconcileEvents(events []api.Event) {
```

Comment:
I think move most of this into events package and keep all the event handling code together.